docs/content/developer/getting-started/move-trace-debug.mdx

```diff
@@ -0,0 +1,51 @@
+---
+description: Guide to install the IOTA Move Trace Debugging Tool.
+title: Debugging Tool
```

Comment:
I rather use only one: frontmatter title or a markdown level 1 header. So I would remove one of those